transcom/mymove
# MB-6437 Add script/data to benchmark doc viewers

## Description

We need to ensure that the document viewer is performant & reliable in
the transportation offices, which are known to have bandwidth issues.
This PR adds a script to populate a single move with any supported files
from the `/pkg/testdatagen/testdata/bandwidth_test_docs` directory. Supported
file types are PNG, JPG, and PDF. Unsupported files will display an error in the console
when running the script, but should not prevent the script from continuing to run. 
This PR includes 3 files of varying sizes, that both the TOO and TIO can view in the 
orders document viewer and proof of service document viewer respectively.

## Reviewer Notes

To run the script:

```
make db_bandwidth_up
```

To verify the docs were created and can be viewed, follow these steps:

1. Start the server and client:

```
make server_run
make office_client_run
```

2. Click on "Local Sign in" when the office app appears
3. Sign in with the single office user that the script created, and
view the docs as a TOO, and then as a TIO.

As a TOO:

1. Click on the only move in your queue
2. Click on "Orders" in the left sidebar
3. Click on "View & edit orders" in the top right

As a TIO:

1. Click on "Change user role" in the top left
2. Click on "Select transportation_invoicing_officer"
1. Click on the only payment request in your queue
2. Click on "Review service items"

Other things to test:
1. Add more files, or replace the files in the `bandwidth_test_docs` directory with files of your choosing.
2.  Verify that all supported file types are uploaded, but that no unsupported files types are uploaded.

Is there anything you would like reviewers to give additional scrutiny?

## Code Review Verification Steps

* Any new migrations/schema changes:

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6437) for this change
